indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -37,4 +37,21 @@
 
         setupListGenerator(filterConfig);
     };
+    global.setupAbstractFeature = function setupAbstractFeature() {
+        $('#enable-abstracts').on('click', function(evt) {
+            evt.preventDefault();
+
+            var $this = $(this);
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                success: function() {
+                    IndicoUI.Dialogs.Util.progress();
+                    location.reload();
+                },
+                error: handleAjaxError
+            });
+        });
+    };
```

Comment:
Hm, right. Then maybe we should allow `data-reload-after` also without an `ajaxdialog`?